contrail-controller/hooks/contrail_controller_hooks.py

```diff
@@ -275,7 +277,45 @@ def update_southbound_relations(rid=None):
         "ssl-enabled": config.get("ssl_enabled") and config.get("config_analytics_ssl_available"),
         # base64 encoded ca-cert
         "ca-cert": config.get("ca_cert"),
+        "cassandra_user": "admin",
```

Comment:
I asked you to implement a three separate functions:
get_cassandra_connection_details, get_zookeeper_connection_details, get_rabbitmq_connection_details
These functions will return specific dicts. These functions will be used to provide details to relation and to get local config. then some can merge all data into single issu.conf